allure-junit5/src/test/java/io/qameta/allure/examples/junit5/AllureQuickStartTest.java

```diff
@@ -7,7 +7,7 @@
 
 import static io.qameta.allure.Allure.step;
 
-public class AllureSimpleTest {
+public class AllureQuickStartTest {
 
     @Test
     @DisplayName("allureSimpleTest displayName")
```

Comment:
Let's change to 
```
@DisplayName("Custom display name instead of default:  allureSimpleTest()")
```
to emphasize the "customization" aspect, and compare it to the default behaviour

Reply:
I will add other proposals in the same thread, because the code below is the "old one", and I can't comment the exact line... 

So...

Let's make the "simpleTestMethod" name more consistent/comparable with "simple lambda step" by the following change:
<img width="1083" alt="image" src="https://user-images.githubusercontent.com/1961118/112053228-c1d2c580-8b5c-11eb-9889-9fd61a4f3c11.png">

Reply:
I would also remove the "test" word in a "test method" term applied to the simpleMethodStep, because (correct me If I am wrong) the "test method" is the term sounding better with methods that are actually test, i.e. annotated with `@Test`... 

<img width="971" alt="image" src="https://user-images.githubusercontent.com/1961118/112053570-2db52e00-8b5d-11eb-90a9-6b514713a694.png">

Reply:
I would also remove the `step"Method parameter:  " + param);` line, because look how it is rendered in the actual report:

<img width="477" alt="image" src="https://user-images.githubusercontent.com/1961118/112053767-6bb25200-8b5d-11eb-960b-f5658d3cd7f4.png">

Here we have two sub-lines 
* param method parameter
* method parameter: method parameter
that are two similar and might confuse when trying to understand what happens from the report...

I believe, here we can just leave it as simple as:

<img width="1125" alt="image" src="https://user-images.githubusercontent.com/1961118/112053724-59381880-8b5d-11eb-9e7b-1960888d4cca.png">

Reply:
I would change the line with step status to 

<img width="1132" alt="image" src="https://user-images.githubusercontent.com/1961118/112054640-7d482980-8b5e-11eb-963a-ef3ddf2684b8.png">

```
step("Simple kind of todo-step with status", Status.SKIPPED);
```

... in order to probably reflect something more usable for average allure new-comer... 

Or even consider removing "step with status" from this context. If this allure feature can be rarely met in real life, we can reflect it in AllureStepTest.java, no need to reflect it here, in "quick start for kind of average users" context.